packages/hms_room_kit/lib/src/common/utility_functions.dart

```diff
@@ -28,7 +29,7 @@ class Utilities {
     }
     List<String>? parts = name.trim().split(" ");
     if (parts.length == 1) {
-      name = parts[0][0];
+      name = parts[0].substring(0, math.min(2, parts[0].length));
```

Comment:
ensure that this does not cause any app crashes when name is small

Reply:
Can you think of any case where this might crash?